### ISSUE TITLE
r.0.11 avro missing files 

### ISSUE THREAD

**Issue by arturovivas** [tensorflow/tensorflow#9130]
Hi

I am trying to install tf r0.11 in a Centos 7 without GPU support from sources. I have a similar problem like the https://github.com/tensorflow/tensorflow/issues/4312. I followed the recommended actions, yet I get the same error. BESIDES, I get the following (not described in the issue 4312) error:

`Error downloading from http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz` 

I tryed to find this file using the web browser and I coudn't find it. The online repository doesn't have it anymore. Just the avro-1.8.1 and avro-1.7.7 are available.

Thanks
Regards
A

**Comment by asimshankar** [comment]
0.11 is a very old release (prior to stability guarantees of 1.0). Is it possible for you to use 1.0?

That said, **please provide all information asked for in the new issue template**. Without that it is hard to be able to help (in particular, do provide the exact sequence of commands to reproduce the problem).

Thanks!

**Comment by arturovivas** [comment]
I run my applications in serveral clusters. One of them, has only tf.11 installed, and since I don't have admin rights, I would like to keep the other servers with the tf.11. 

The secuence of commands was:

```
./configure
~/tensorflow ~/tensorflow
Please specify the location of python. [Default is /home/AVV/rocm/anaconda2/bin/python]: 
Do you wish to build TensorFlow with Google Cloud Platform support? [y/N] N
No Google Cloud Platform support will be enabled for TensorFlow
Do you wish to build TensorFlow with Hadoop File System support? [y/N] N
No Hadoop File System support will be enabled for TensorFlow
Found possible Python library paths:
  /home/AVV/rocm/anaconda2/lib/python2.7/site-packages
Please input the desired Python library path to use.  Default is [/home/AVV/rocm/anaconda2/lib/python2.7/site-packages]

/home/AVV/rocm/anaconda2/lib/python2.7/site-packages
Do you wish to build TensorFlow with GPU support? [y/N] N
No GPU support will be enabled for TensorFlow
Configuration finished
...................
INFO: Starting clean (this may take a while). Consider using --expunge_async if the clean takes more than several minutes.
..........
WARNING: /home/AVV/rocm/.cache/bazel/_bazel_rocm/5fee00a38fb04f0267978a3cccec9960/external/protobuf/protobuf.bzl:90:19: Variables HOST_CFG and DATA_CFG are deprecated in favor of strings "host" and "data" correspondingly.
WARNING: /home/AVV/rocm/.cache/bazel/_bazel_rocm/5fee00a38fb04f0267978a3cccec9960/external/protobuf/protobuf.bzl:96:28: Variables HOST_CFG and DATA_CFG are deprecated in favor of strings "host" and "data" correspondingly.
ERROR: /home/AVV/rocm/tensorflow/tensorflow/contrib/avro/BUILD:13:1: no such package '@avro_archive//': Error downloading from http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz to /home/AVV/rocm/.cache/bazel/_bazel_rocm/5fee00a38fb04f0267978a3cccec9960/external/avro_archive: Error downloading http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz to /home/AVV/rocm/.cache/bazel/_bazel_rocm/5fee00a38fb04f0267978a3cccec9960/external/avro_archive/avro-cpp-1.8.0.tar.gz: 404 Not Found: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz was not found on this server.</p>
</body></html>
 and referenced by '//tensorflow/contrib/avro:example_h'.
ERROR: /home/AVV/rocm/tensorflow/tensorflow/contrib/avro/BUILD:13:1: no such package '@avro_archive//': Error downloading from http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz to /home/AVV/rocm/.cache/bazel/_bazel_rocm/5fee00a38fb04f0267978a3cccec9960/external/avro_archive: Error downloading http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz to /home/AVV/rocm/.cache/bazel/_bazel_rocm/5fee00a38fb04f0267978a3cccec9960/external/avro_archive/avro-cpp-1.8.0.tar.gz: 404 Not Found: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz was not found on this server.</p>
</body></html>
 and referenced by '//tensorflow/contrib/avro:example_h'.
ERROR: Evaluation of query "deps((//tensorflow/... union @bazel_tools//tools/jdk:toolchain))" failed: errors were encountered while computing transitive closure.
```

**Comment by asimshankar** [comment]
I see, it seems that `avro-1.8.0` is no longer downloadable. Your best bet would be to [update `workspace.bzl`](https://github.com/tensorflow/tensorflow/blob/r0.11/tensorflow/workspace.bzl#L204) with the compatible AVRO version - perhaps to http://www-us.apache.org/dist/avro/avro-1.8.1/cpp/avro-cpp-1.8.1.tar.gz (and update the checksum accordingly).

Please feel free to fix that up and send a pull request to the r0.11 branch if that works.

Thanks!

**Comment by itsmeolivia** [comment]
Automatically closing due to lack of recent activity. Since this issue is old at this point, please reopen the issue if it still occurs when tried with the latest version of Tensorflow. Thank you.

**Comment by CriMenghini** [comment]
I'm trying to install from source TensorFlow r.011 (since I need to use a code developed using this version). 

To fix the error you should do the following change:

*Original snippet from `workspace.bzl`*:

`native.new_http_archive(
    name = "avro_archive",
    url = "http://www-us.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz",
    sha256 = "ec6e2ec957e95ca07f70cc25f02f5c416f47cb27bd987a6ec770dcbe72527368",
    strip_prefix = "avro-cpp-1.8.0",
    build_file = str(Label("//:avro.BUILD")),
  )
`

*Modified one*:

`native.new_http_archive(
    name = "avro_archive",
    url = "https://archive.apache.org/dist/avro/avro-1.8.0/cpp/avro-cpp-1.8.0.tar.gz",
    sha256 = "ec6e2ec957e95ca07f70cc25f02f5c416f47cb27bd987a6ec770dcbe72527368",
    strip_prefix = "avro-cpp-1.8.0",
    build_file = str(Label("//:avro.BUILD")),
  )`

Basically, the url changes.

Hope this help.
